ts/editor/editor-toolbar/OptionsButton.svelte

```diff
@@ -0,0 +1,49 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import CheckBox from "../../components/CheckBox.svelte";
+    import DropdownItem from "../../components/DropdownItem.svelte";
+    import IconButton from "../../components/IconButton.svelte";
+    import Popover from "../../components/Popover.svelte";
+    import WithFloating from "../../components/WithFloating.svelte";
+    import { bridgeCommand } from "../../lib/bridgecommand";
+    import * as tr from "../../lib/ftl";
+    import { shrinkImagesByDefault } from "../image-overlay/ImageOverlay.svelte";
+    import { cogIcon } from "./icons";
+
+    let showFloating = false;
+
+    function toggleShrinkImages(_evt: MouseEvent): void {
+        $shrinkImagesByDefault = !$shrinkImagesByDefault;
+        bridgeCommand("toggleShrinkImages");
+        showFloating = false;
+    }
+</script>
+
+<WithFloating
+    show={showFloating}
+    placement="bottom"
+    inline
+    on:close={() => (showFloating = false)}
+    let:asReference
+>
+    <span use:asReference>
+        <IconButton
```

Comment:
I think `<IconButton slot="reference">` should work just fine here

Reply:
Dang, made the change but forgot to commit it :sweat_smile:. Pushed a follow-up commit.